src/main/java/de/qabel/desktop/ui/contact/ContactController.java

```diff
@@ -107,12 +102,9 @@ public void initialize(URL location, ResourceBundle resources) {
     }
 
     private void createButtonGraphics() {
-        Image importGraphic = new Image(getClass().getResourceAsStream("/img/import_black.png"));
-        importButton.setGraphic(new ImageView(importGraphic));
-        Tooltip.install(importButton, new Tooltip(resourceBundle.getString("contactImport")));
-        Image exportGraphic = new Image(getClass().getResourceAsStream("/img/export.png"));
-        exportButton.setGraphic(new ImageView(exportGraphic));
-        Tooltip.install(exportButton, new Tooltip(resourceBundle.getString("contactExport")));
+        searchButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/icon/search.png"))));
+        contactsButton.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/account_multiple.png"))));
+        contactMenu.setGraphic(new ImageView(new Image(getClass().getResourceAsStream("/img/dots_vertical.png"))));
```

Comment:
Where have all the tooltips gone?
Btw: where are the tooltips for the `IdentityContextMenuController`?